### PR TITLE
[TT-40] Expand output path with tilde

### PR DESCRIPTION
**Pull request by medric**
Node doesn't expand `~`to the current user's home directory - see https://github.com/nodejs/node/issues/684. 

Looks like this has to be implemented manually if we really want to leverage this feature, hence the following changes.

### CODE REVIEW

**Review by lfportal — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -255,7 +255,8 @@ OPTIONS
   -h, --help                   show CLI help
   -p, --port=port              (required) [default: 3010] Port on which to run the mock server
   --pathPrefix=pathPrefix      Prefix to prepend to each endpoint path
-  --proxyBaseUrl=proxyBaseUrl  If set, the server would act as a proxy and fetch data from the given remote server
+
+  --proxyBaseUrl=proxyBaseUrl  If set, the server will act as a proxy and fetch data from the given remote server
```
Comment (fwouts) [inline README.md:259]: This seems unrelated?

File: lib/src/utilities/expand-path-with-tilde.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import os from "os";
+
+export default function expandPathWithTilde(path: string) {
```
Comment (fwouts) [inline lib/src/utilities/expand-path-with-tilde.ts]: Consider using a named export to follow conventions in the Spot codebase.

**Review by fwouts — APPROVED** [review]
